pkgs/development/python-modules/dmt-core/default.nix

```diff
@@ -0,0 +1,89 @@
+{
+  buildPythonPackage,
+  colormath,
+  cycler,
+  fetchPypi,
+  h5py,
+  joblib,
+  lib,
+  more-itertools,
+  numpy,
+  pandas,
+  pint,
+  pyarrow,
+  pytestCheckHook,
+  python,
+  pyyaml,
+  reprint,
+  requests,
+  scikit-rf,
+  scipy,
+  semver,
+  setuptools,
+  stdenv,
+  verilogae,
+}:
+
+buildPythonPackage rec {
+  pname = "dmt-core";
+  version = "2.1.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "DMT_core";
+    hash = "sha256-489E+uNn4NgyCwxsUMEPH/1ZuM+5uNq4zx8F88rkHMU=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    colormath
+    cycler
+    h5py
+    joblib
+    more-itertools
+    pandas
+    pint
+    pyarrow
+    pyyaml
+    requests
+    scikit-rf
+    scipy
+    setuptools
+    numpy
+    semver
+  ];
+
+  nativeBuildInputs = [
+    reprint
+    verilogae
+  ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [
+    "DMT.core"
+    "reprint"
+    "verilogae"
+  ];
+
+  LD_LIBRARY_PATH = "${stdenv.cc.cc.lib}/lib/";
```

Comment:
You are sure we cannot use patchelf here?

Reply:
we cannot :)